### PR TITLE
[IMP] base_user_role: Adding a new tab for user comments in roles form.

### PR DESCRIPTION
**Pull request by HaraldPanten**
As it is in V12.

It's useful as many times users like to have a summary of the rules, or any other changes.

### CODE REVIEW

**Comment by HaraldPanten** [comment]
@pedrobaeza do you mind merging that easy one, to keep consistancy in newer versions? By the way, should I open another PR for V14?

THX

File: base_user_role/views/role.xml [hunk 1]
```diff
@@ -34,6 +34,9 @@
                                 </tree>
                             </field>
                         </page>
+                        <page name="role_inernal_notes" string="Internal Notes">
```
Comment (pedrobaeza) [inline base_user_role/views/role.xml]: Typo and `role` is redundant. Better to prefix it with the type of element:

```suggestion
                        <page name="page_internal_notes" string="Internal Notes">
```

**Comment by HaraldPanten** [comment]
@pedrobaeza done

File: base_user_role/views/role.xml [hunk 2]
```diff
@@ -34,6 +34,9 @@
                                 </tree>
                             </field>
                         </page>
+                        <page name="page_inernal_notes" string="Internal Notes">
```
Comment (pedrobaeza) [inline base_user_role/views/role.xml]:
```suggestion
                        <page name="page_internal_notes" string="Internal Notes">
```

**Comment by HaraldPanten** [comment]
OMG... sorry Pedro.

You know that "haste makes waste"!

**Review by pedrobaeza — APPROVED** [review]

**Comment by HaraldPanten** [comment]
Should I do that for V14 as well?

**Pull request merged**